doc/api/cli.md

```diff
@@ -1663,6 +1648,25 @@ Disable the `node-addons` exports condition as well as disable loading
 native addons. When `--no-addons` is specified, calling `process.dlopen` or
 requiring a native C++ addon will fail and throw an exception.
 
+### `--no-async-context-frame`
+
+<!-- YAML
+added: v22.7.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55552
+    description: Marked as stable.
+-->
+
+> Stability: 2 - Stable
+
+Enables the use of [`AsyncLocalStorage`][] backed by `AsyncContextFrame` rather
+than the default implementation which relies on async\_hooks. This new model is
+implemented very differently and so could have differences in how context data
+flows within the application. As such, it is presently recommended to be sure
+your application behaviour is unaffected by this change before using it in
+production.
```

Comment:
This description is incorrect, no? This flag **disables** the feature, not enables it.

Reply:
Thanks for catching that. I've updated it now to reflect the true state.